### DISCUSSION TITLE
Minimum hardware requirements for deployment?

### DISCUSSION THREAD

**Discussion by johannesschaffer** [payloadcms/payload#1683] [General]
I need to choose a Digital Ocean droplet plan. Is the base tier 4$ enough, and when not, which plan/hardware is at least required?
Strapi specifies 1 core, 2gb RAM, but that seems quite high?

**Comment by FrancocDev** [comment]
I'm running payload on railway and it's really far from the pay play. I'm using about 0.2 ram on standby and 0.5 when recieve more visits.

**Comment by johannesschaffer** [comment]
Thanks for sharing your experience. I since deployed it to the DigitalOcean App Platform and running on the basic tear (0.5GB RAM). Seems to work great 👌

**Reply by jamesvclements** [reply]
Did you have any issues with cross-end in the serve command when deploying on App Platform? I get "cross-env not found" even though it's in my dependency list.

**Reply by johannesschaffer** [reply]
Yeah, you either need to reinstall the dev-dependencies, since DO uninstalls them, or remove the cross-env command (only required in Windows)
